lib/honeycomb/integrations/rack.rb

```diff
@@ -13,8 +13,11 @@ module Rack
       ["HTTP_VERSION", "request.http_version"],
       ["HTTP_HOST", "request.host"],
       ["REMOTE_ADDR", "request.remote_addr"],
+      ["HTTP_ACCEPT", "request.header.accept"],
+      ["HTTP_CONTENT_TYPE", "request.header.content_type"],
```

Comment:
Can we add all of these new fields to https://github.com/honeycombio/beeline-ruby/blob/master/spec/support/event_data_shared_examples.rb#L17?